docker-compose.yml

```diff
@@ -11,5 +11,7 @@ web:
     SERVER_APP_NAME: docker-test
     REPLACE_NEWRELIC_APP: abcdefg
     REPLACE_NEWRELIC_LICENSE: hijklmno
+    S6_KILL_FINISH_MAXTIME: 1
+    S6_KILL_GRACETIME: 1
```

Comment:
dev users will probably want these environment variables set in their compose file to make stack takedown really fast